### PR TITLE
Switch from Godeps to Glide

### PR DESCRIPTION
**Pull request by colemickens**
Let me know if you want me to hold off on this for sequencing with other changes.

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- package: github.com/Azure/azure-sdk-for-go
+  subpackages:
+  - /arm/examples/helpers
+  - arm/storage
+  - core/http
+  - core/tls
+  - management
+  - management/location
+  - management/virtualmachine
+  - management/virtualmachinedisk
```
Comment (ahmetalpbalkan) [inline glide.yaml]: any ideas why this list is incomplete? (I'm not familiar to glide, it well may be because integration tests depend on these)

Comment (colemickens) [inline glide.yaml]: I need to take a look at this more closely, I don't know why it's even listing itself in here.

**Comment by brendandixon** [comment]
LGTM

**Comment by ahmetalpbalkan** [comment]
are we gonna merge changes to the arm/** ? I thought they were going to happen next time we generate the code.

**Comment by colemickens** [comment]
As I mentioned, I can hold this until we regenerate. It wouldn't be a bad idea since it would give us a chance to update the other dependencies as well. Either way is fine, I have the "convert to glide" process scripted now.

**Comment by colemickens** [comment]
(It would be okay to merge it, since I've pinned the deps at the same version they're currently with Godeps.)

**Comment by brendandixon** [comment]
@garimakhuble is actively rebuilding the arm/* packages.

**Comment by brendandixon** [comment]
Oops. @garimakhulbe

**Comment by jen20** [comment]
@brendandixon do you have an ETA on the new packages? Currently any consumer of the ARM API is completely broken.

**Comment by stack72** [comment]
FWIW, this is the error:

```
github.com/Azure/azure-sdk-for-go (download)
package github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest: cannot find package "github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest" in any of:
	/opt/boxen/homebrew/Cellar/go/1.5.3/libexec/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest (from $GOROOT)
	/Users/stacko/Code/go/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest (from $GOPATH)
package github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure: cannot find package "github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure" in any of:
	/opt/boxen/homebrew/Cellar/go/1.5.3/libexec/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure (from $GOROOT)
	/Users/stacko/Code/go/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/azure (from $GOPATH)
package github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/date: cannot find package "github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/date" in any of:
	/opt/boxen/homebrew/Cellar/go/1.5.3/libexec/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/date (from $GOROOT)
	/Users/stacko/Code/go/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/date (from $GOPATH)
package github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/to: cannot find package "github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/to" in any of:
	/opt/boxen/homebrew/Cellar/go/1.5.3/libexec/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/to (from $GOROOT)
	/Users/stacko/Code/go/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/Azure/go-autorest/autorest/to (from $GOPATH)
package github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/dgrijalva/jwt-go: cannot find package "github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/dgrijalva/jwt-go" in any of:
	/opt/boxen/homebrew/Cellar/go/1.5.3/libexec/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/dgrijalva/jwt-go (from $GOROOT)
	/Users/stacko/Code/go/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/github.com/dgrijalva/jwt-go (from $GOPATH)
package github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12: cannot find package "github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12" in any of:
	/opt/boxen/homebrew/Cellar/go/1.5.3/libexec/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12 (from $GOROOT)
	/Users/stacko/Code/go/src/github.com/Azure/azure-sdk-for-go/Godeps/_workspace/src/golang.org/x/crypto/pkcs12 (from $GOPATH)
```

**Comment by ahmetalpbalkan** [comment]
@jen20 @stack72 at no point, you should be pulling directly from master. Please stick to a tag or a specific commit number you know that is working fine for you.

**Comment by brendandixon** [comment]
@stack72 @jen20 I agree with @ahmetalpbalkan. However, it is our intention to get those out today.

**Pull request merged**